api/openapi.yaml

```diff
@@ -30,228 +30,654 @@ tags:
+        '200':
+          $ref: '#/components/responses/Ok'
+        '400':
+          $ref: '#/components/responses/NotFound'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
+        default:
+          $ref: '#/components/responses/DefaultResponse'
+      summary: "Update a place's data."
+      tags:
+        - stalker web app
+  '/organization/{organizationId}/places':
+    parameters:
+      - $ref: '#/components/parameters/organizationId'
+    post:
+      description: Add a new place to the given organization. Operation reserved only to the managers and up.
+      operationId: postPlaceByOrganizationId
+      parameters:
+        - $ref: '#/components/parameters/organizationId'
+      requestBody:
         content:
           application/json:
             schema:
-              $ref: ./model/Login.v1.yaml
+              $ref: ./model/PlaceData.v1.yaml
+        required: true
       responses:
         '200':
-          description: Ok.
           content:
             application/json:
               schema:
-                $ref: ./model/User.v1.yaml
+                $ref: ./model/GenericId.v1.yaml
+          description: Ok. The response contains the id of the newly created place.
         '401':
           $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
         default:
           $ref: '#/components/responses/DefaultResponse'
-      summary: Create a login request for an user
-  '/users/{userId}/organizations/connected':
-    parameters:
-      - $ref: '#/components/parameters/userId'
-    get:
+      summary: Create a new place.
       tags:
-        - stalker mobile app
         - stalker web app
-      description: Get the list of organizations that a user has connected to.
-      operationId: getSpecificUserSubscriptions
+  '/organization/{organizationId}/users/connections':
+    get:
+      description: Get the list of all users connected to an organization.
+      operationId: get-organization-organizationId-users-connected
       responses:
         '200':
-          description: The list of the ids of the organizations the user is connected to.
           content:
             application/json:
               schema:
-                type: object
                 properties:
-                  connected:
-                    type: array
+                  connectedUsers:
                     items:
-                      $ref: ./model/GenericId.v1.yaml
-                required:
-                  - connected
+                      $ref: ./model/User.v1.yaml
+                    type: array
+                type: object
+          description: OK
+        default:
+          $ref: '#/components/responses/DefaultResponse'
+      summary: Read the list of users connected to an organization.
+      tags: []
+    parameters:
+      - $ref: '#/components/parameters/organizationId'
+  '/organization/{organizationId}/users/role':
```

Comment:
qua serve un livello in più per determinare a quale utente tolgo i privilegi, perché nel delete almeno mi pare non si può inviare un body

Reply:
si può ma non è raccomandato quindi hai ragionissima; cambiamo in 
/organization/{organizationId}/user/{userId}/role. proabilmente cambio anche le altre per coerenza